4.1/crud-operation-fetch.md

```diff
@@ -87,6 +87,27 @@ public function fetchUser() {
     }
 ```
 
+## Using FetchOperation with `select2_ajax` filter
```

Comment:
```suggestion
<a name="fetch-ajax-filter"></a>
## Using FetchOperation with `select2_ajax` filter
```

Reply:
Hmm... I wouldn't put these here. I think it'd be better if we put this inside the filter directly. As an either/or. For example: _as the source of your filter, you can either (A) use the FetchOperation or (B) code your own route, and controller method that responds with results_. Then detailing steps for the two sources of input.

The reason I'm thinking this is because... I don't think people will come to the FetchOperation and think "_I want to use this in a filter_". No, what's more likely is that they'll go _to the filters docs_ to add one, and when they get to `select2_ajax` they expect it to work with their existing FetchOperation. At least that's what I think.

So I think moving these instructions to filters will:
- make them available where the user is looking for them
- keep the FetchOperation docs isolated from where it's used (fields and filters); though we _could_ link to instructions to the instructions in both fields and filters, just in case people come here first;